projects/lights_control/src/lights_blinker.c

```diff
@@ -0,0 +1,60 @@
+#include <stddef.h>
+
+#include "event_queue.h"
+#include "soft_timer.h"
+#include "status.h"
+
+#include "lights_blinker.h"
+
+static void prv_timer_callback(SoftTimerID timer_id, void *context) {
+  LightsBlinker *blinker = (LightsBlinker *)context;
+  blinker->state = !blinker->state;
+  event_raise(blinker->event_id, (uint16_t)blinker->state);
```

Comment:
Probably want to make sure your `lights_gpio` module has this data field documented.

Reply:
yes, documentation has been added 👍